h/templates/client/settings/account.html

```diff
@@ -1,5 +1,51 @@
 <div class="tab-pane" title="Account">
   <form class="account-form form"
+        name="changeEmailForm"
+        ng-submit="changeEmail(changeEmailForm)"
+        novalidate form-validate>
+
+    <h2 class="form-heading"><span>Change Your Email Address</span></h2>
+
+    <div class="form-field">
+      <label class="form-label" for="field-email">Email Address:</label>
+      <input id="field-email" class="form-input" type="email" name="email" placeholder="{{email}}" required ng-model="changeEmail.email" />
```

Comment:
I'm not keen on the placeholder. I find it super confusing to interact with:

![placeholder-badness](https://cloud.githubusercontent.com/assets/3602/7726233/deca6ff6-ff00-11e4-9e82-cdbfa113216a.gif)

Either set default values (i.e. the value attribute) or just skip this entirely.